sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeContentOptions.cs

```diff
@@ -38,10 +38,17 @@ public RecognizeContentOptions()
         public string Language { get; set; }
 
         /// <summary>
+        /// <para>
         /// Custom page numbers for multi-page documents(PDF/TIFF). Input the number of the
         /// pages you want to get OCR result. For a range of pages, use a hyphen.
         /// Separate each page or range with a comma.
+        /// For sample "1-3", {"1", "3", "5-7"}.
```

Comment:
Not sure I understand the sample for "1-3". why is "5-7" included?

Reply:
updated